nilearn/decomposition/dict_learning.py

```diff
@@ -291,15 +291,16 @@ def _raw_fit(self, data):
 
         dict_init = self.loadings_init_
 
-        n_iter = ((n_features - 1) // self.batch_size + 1) * self.n_epochs
+        max_iter = ((n_features - 1) // self.batch_size + 1) * self.n_epochs
 
         if self.verbose:
             print("[DictLearning] Learning dictionary")
+        # TODO: turn n_iter to max_iter when dropping python 3.7
```

Comment:
probably worth opening an issue to track some of the things we may want to check when dropping python 3.7

Reply:
It doesn't hurt, but we can also open a new milestone for release 0.11 and document there because that will be when we drop 3.7